templates/composable_roles_overcloud_ceph_deploy.sh.j2

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+openstack overcloud ceph deploy -v \
+-o /home/stack/templates/overcloud-ceph-deployed.yaml \
+--container-image-prepare "{{ containers_home_dir }}/containers-prepare-parameter.yaml" \
+{% if install.ceph.initial.conf.file %} --config {{ ansible_user_dir }}/initial-ceph.conf{% endif %} \
+{% if install.ceph.osd.spec.file %} --osd-spec {{ ansible_user_dir }}/osd-spec.yml{% endif %} \
+{% if ceph_node_cnt|int < 3 %}--single-host-defaults {% endif %} \
+--stack "{{ install.overcloud.stack }}" \
+{% if install.ceph.cluster.name and install.ceph.cluster.name|length > 0 %}--cluster {{ install.ceph.cluster.name }}{% endif %} \
+--network-data "{{ template_base }}/network/network_data_v2.yaml" \
+--roles-data "/home/stack/roles_data.yaml" \
```

Comment:
can you make the necessary changes in the roles_data_path in the code instead of maintaining a duplicate of [overcloud_ceph_deploy.sh.j2](https://github.com/redhat-openstack/infrared/blob/master/plugins/tripleo-overcloud/templates/overcloud_ceph_deploy.sh.j2)

Reply:
We already generate roles_data on the undercloud, so the idea was to to just use it from the undercloud node like it is being done for composable roles currently.
The other way around, we'll have to copy the generated roles_data file from undercloud to jumphost and then override infrared. This seems to be a simple & similar approach to what we are already doing, hence went with this.

Reply:
If you update --roles-path  to ~/virt/roles (as it is in infrared) and other references of roles path in the jetpack code that should be sufficient no need to maintain a duplicate of [overcloud_ceph_deploy.sh.j2](https://github.com/redhat-openstack/infrared/blob/master/plugins/tripleo-overcloud/templates/overcloud_ceph_deploy.sh.j2).  Let me know if still have any doubts

Reply:
Made the amendments